### ISSUE TITLE
GENEVE not supported on Raspberry Pi 4 image?

### ISSUE THREAD

**Issue by ruifung** [siderolabs/talos#7314]
## Bug Report
When booting on a raspberry pi 4, GENEVE encapsulation does not appear to be supported.
### Description
While randomly deciding to add a spare raspberry pi 4 to my existing cluster running cilium, with the geneve tunnel encapsulation, cilium agent fails to start.
### Logs
![image](https://github.com/siderolabs/talos/assets/1073945/ca0580f2-e82a-44de-8bfb-38d512e012ba)

### Environment

- Talos version: [`talosctl version --nodes <problematic nodes>`]
```
Client:                                                                                                                                                                                                                                                                                         Tag:         v1.4.4                                                                                                                                                                                                                                                                     SHA:         168114fc
        Built:
        Go version:  go1.20.4
        OS/Arch:     windows/amd64
Server:
        NODE:        pathweb-piworker-1.vsvc.home.arpa
        Tag:         v1.4.5
        SHA:         45679b0e
        Built:
        Go version:  go1.20.4
        OS/Arch:     linux/arm64
        Enabled:     RBAC
```
- Kubernetes version: [`kubectl version --short`]
```
Client Version: v1.25.9
Kustomize Version: v4.5.7
Server Version: v1.26.5
```
- Platform:
Raspberry Pi 4B, 4GB

**Comment by smira** [comment]
Looks like GENEVE was enabled for amd64, but not arm64: https://github.com/siderolabs/pkgs/pull/278

We'll fix that for the next release of Talos.

**Comment by smira** [comment]
should be fixed Talos >= 1.5.0
